pkg/client/password_test.go

```diff
@@ -0,0 +1,21 @@
+package client
+
+import (
+	"testing"
+)
+
+func TestCheckPassword(t *testing.T) {
+	var testCases = []struct {
+		pass string
+		ok   bool
+	}{
+		{"12345678", true},
+		{"test", false},
+	}
+
+	for _, tc := range testCases {
+		if err := checkPassword(tc.pass); err != nil && tc.ok {
+			t.Errorf("expected failure, got success")
```

Comment:
💅  I'm bit confused here, please don't get me wrong, right?
What do you think about changing for something like this:
```
if err := checkPassword(tc.in); (err != nil) && tc.expectError {
    t.Errorf("expected error: %v, got error %v", tc.expectError, err)
} 
```
💅